app/src/main/res/values/strings.xml

```diff
@@ -23,7 +23,7 @@
   <string name="coc_long_version_subtitle">Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.\n\nParticipants asked to stop any harassing behavior are expected to comply immediately.\n\nIf a participant engages in behavior that violates this code of conduct, the conference organizers may take any action they deem appropriate, including warning the offender or expulsion from the conference with no refund.</string>
 
   <string name="coc_contact_information_title">Contact Information</string>
-  <string name="coc_contact_information_subtitle">If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a self.conference staff member or volunteer immediately.\n\nYou may also contact any volunteer and ask to be put in touch with the conference chairs, Amber Conville &amp; Erika Carlson.\n\nIf the matter is especially urgent, please contact conference co-chair Erika Carlson at (313) 473-7560.\n\nConference staff will be happy to help participants contact hotel/venue security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the conference. We value your attendance.\n\nAdapted from http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy</string>
+  <string name="coc_contact_information_subtitle">If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a self.conference staff member or volunteer immediately.\n\nYou may also contact any volunteer and ask to be put in touch with the conference chair, Amber Conville.\n\nIf the matter is especially urgent, please contact conference co-chair Amber Conville at (734) 660-0754.\n\nConference staff will be happy to help participants contact hotel/venue security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the conference. We value your attendance.\n\nAdapted from http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy</string>
```

Comment:
Thanks for updating this. For 2016, perhaps it makes sense to pull this down from the API instead of hardcoding it into the app. I think we ran out of time last year to visit this aspect of things. What do you think?